source/extensions/dynamic_modules/sdk/go/abi.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
```

Comment:
note: this is the hard-copy of the original abi.h, not the symlink to it. In order to enable the off-tree development, this is necessary and explained in the README.md.